packages/react-native-gesture-handler/apple/Handlers/RNNativeViewHandler.mm

```diff
@@ -148,6 +148,11 @@ - (void)bindToView:(UIView *)view
     [control addTarget:self action:@selector(handleDragExit:forEvent:) forControlEvents:UIControlEventTouchDragExit];
     [control addTarget:self action:@selector(handleDragEnter:forEvent:) forControlEvents:UIControlEventTouchDragEnter];
     [control addTarget:self action:@selector(handleTouchCancel:forEvent:) forControlEvents:UIControlEventTouchCancel];
+
+    if (self.actionType == RNGestureHandlerActionTypeNativeDetector) {
+      [view addGestureRecognizer:_recognizer];
+    }
```

Comment:
Why do we need the dummy recognizer on ui controls? It should do nothing unless attached to a scroll view.

Reply:
In `GestureHandler` base class we have `findViewForEvents` method that uses `self.recognizer.view`. If we do not set it here, buttons won't work as the view for events is `nil`.